app/services/user_service.rb

```diff
@@ -6,12 +6,22 @@ def self.find_user(google_id)
   end
 
   def self.search(email)
-    response = DatabaseService.conn.get("/api/v1/users?by_email=#{email}")
+    response = DatabaseService.conn.get("/api/v1/users?email=#{email}")
     JSON.parse(response.body, symbolize_names: true)
   end
 
   def self.friends
-    response = DatabaseService.conn.get("api/v1/friends")
+    response = DatabaseService.conn.get('api/v1/friends')
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def self.create(user)
+    response = DatabaseService.conn.post 'api/v1/users' do |req|
+      req.headers[:google_id] = user.google_id
+      req.headers[:name] = user.name
+      # .gsub(' ','_')
+      req.headers[:email] = user.email
+    end
```

Comment:
this might be doing the same thing as the `find_or_create_by_auth(auth_data) method I have in the User model. This actually might be better to keep so we don't risk duplicating users on the frontend and backend